### ISSUE TITLE
Use international standards for logic operators

### ISSUE THREAD

**Issue by Adrienrougny** [sbgn/process-descriptions#186]

**Comment by hasanbalci** [comment]
What do we mean here by using international standards?

**Comment by draeger** [comment]
Yes, please do! My personal preference was always to use either mathematical symbols, i.e., ∧, ∨, ⊻, ¬ or symbols from common programming languages, i.e., ``&&``, ``||``, ``!=``, ``!`` or similar. From my perspective, it is not a good choice to have English words in those symbols as it is currently.

**Comment by draeger** [comment]
Mathematics is the most common language.

**Comment by adrienrougny** [comment]
DIscussions at COMBINE: make a vote with clear propositions.

**Comment by cannin** [comment]
If the vote fails, then the change would be for Level 2
